### PR TITLE
Feat continuous play

### PR DESCRIPTION
**Pull request by mixcas**
Plays videos throughout the `list`

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -19,6 +19,7 @@ function scripts_and_styles_method() {
     'siteUrl' => home_url(),
     'themeUrl' => get_template_directory_uri(),
     'isAdmin' => $is_admin,
+    'wp_debug' => WP_DEBUG,
```
Comment (varffs) [inline functions.php:22]: this should go in the starter pack

File: src/js/main.js [hunk 2]
```diff
@@ -141,27 +155,61 @@ Site.Player = {
...
+    if (_this.list.length > currentVideo + 1) {
+
+      // Get next video ID
+      var nextVideo = _this.list[currentVideo + 1];
+
+      // Play next video
+      _this.playVideo(nextVideo);
+    } else {
+      // TODO Close video
+      console.log('No more videos');
```
Comment (varffs) [inline src/js/main.js:212]: should this loop?

File: src/js/main.js [hunk 3]
```diff
@@ -122,6 +122,11 @@ Site.Player = {
   init: function() {
     var _this = this;
 
+    // If WP_DEBUg turn on controls cuz happy Dev :)
```
Comment (varffs) [inline src/js/main.js:125]: very nice

**Review by varffs — COMMENTED** [review]
This is nice. One question really which is about UX but going to merge as we can solve that from master

File: functions.php (hunk 1 above)
Comment (mixcas) [reply functions.php:22]: word

File: src/js/main.js (hunk 2 above)
Comment (mixcas) [reply src/js/main.js:212]: I don't think so. This behavior is something we should talk about and define.
Should it loop?
Should it close the video?
Should it keep playing thru the excluded videos and reset the filters?
What do?

Made a card in trello :)

**Pull request merged**